boot/zephyr/flash_map_extended.c

```diff
@@ -15,14 +15,13 @@
 
 #include "bootutil/bootutil_log.h"
 
-/*
- * For now, we only support one flash device.
- *
- * Pick the SoC Flash driver ID.
- */
+#if defined(DT_FLASH_DEV_NAME)
```

Comment:
```suggestion
#if !defined(CONFIG_XTENSA) && defined(DT_FLASH_DEV_NAME)
...
#elif defined(CONFIG_XTENSA) && defined(DT_SPI_NOR_DRV_NAME)

...
#else
#error 
#endif
```

Reply:
Reason behind suggestion is that we expect coexistence a few flash driver in mcuboot soon.

Reply:
Done.